File: SoftUni Lessons/Python Development/Python Fundamentals May 2023/11. Functions - Exercise/02. Add and Subtract.py
def add_and_subtract(a, b, c):

    def sum_numbers(a, b):
        return a + b

    def subtract(result, c):
        return result - c

    return subtract(sum_numbers(a, b), c)


print(add_and_subtract(int(input()), int(input()), int(input())))

File: SoftUni Lessons/JavaScript Development/JS Fundamentals May 2023/11. Functions - Exercise/02. Add and Subtract.js
function addAndSubtract(a, b, c) {
    function add(a, b) {
        return a + b;
    }

    function subtract(result, c) {
        return result - c;
    }

    console.log(subtract(add(a, b), c));
}


addAndSubtract(23, 6, 10);  // 19